vendor/gopkg.in/ldap.v2/LICENSE

```diff
@@ -0,0 +1,22 @@
+The MIT License (MIT)
```

Comment:
I'd prefer that we used the github hosted version of the package. Given we use vendoring, and go modules just around the corner, using gopkg.in makes little sense.

Reply:
This thing also imports `gopkg.in/asn1-ber.v1` so while I agree in principle I suspect the fight is lost on this package